config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
   # AUTH STARTS
-  match 'auth/:provider/callback', to: 'home#callback', via: [:get, :post]
+  # match 'auth/:provider/callback', to: 'home#callback', via: [:get, :post]
```

Comment:
👃

Reply:
Wanted to know if this route is actually used? I couldn't trace back the controller for it

Reply:
Adding this back, yields this error, because the `home_controller.rb` does not exist and it overrides the routes created by devise token auth

<img width="1239" alt="image" src="https://user-images.githubusercontent.com/18097732/215307731-633ca278-e24c-4ae4-84bf-be897ac20d3f.png">


@sojan-official @tejaswinichile let me know if you have an idea about this

Reply:
@scmmishra I think this has been initially added as a provider for Facebook messenger or Twitter. But we introduced Twitty later, and this endpoint is not used for Facebook. And as you said, the home_controller is also not present in the system.

But to make sure, please deploy this branch on staging, and check if the Facebook and twitter integrations are working. Or let me know; my Facebook account is already configured on staging.

Reply:
Noted, I will test it locally with ngrok

Reply:
@tejaswinichile is there any docs on setting up facebook and twitter locally?